codec/address.go

```diff
@@ -21,10 +21,28 @@ const (
 	maxBech32Size = 90
 )
 
-type Address [AddressLen]byte
+type (
+	LID     [AddressLen]byte // Long ID
+	Address LID
+)
 
 var EmptyAddress = [AddressLen]byte{}
```

Comment:
Once creating LID, you should not bypass it like this when aliasing? I think you'd want to do `EmptyAddress = LID{}`?